src/main/java/br/com/invillia/lyon/userevents/response/UserResponse.java

```diff
@@ -0,0 +1,23 @@
+package br.com.invillia.lyon.userevents.response;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+@Getter
+@Setter
+@Component
```

Comment:
De novo você adicionou uma classe de domínio no contexto do Spring????

Reply:
Força do habito acredito toda classe que vou criando vou colocando pra ela ser gerenciada pelo spring mas nesse caso não precisava mesmo